src/notarytool.ts

```diff
@@ -64,6 +64,31 @@ async function getNotarizationLogs(opts: NotarizeOptions, id: string) {
   }
 }
 
+export function parseNotarytoolOutput(output: string): any {
+  const rawOut = output.trim();
+
+  let jsonOut: string = '';
+
+  for (const line of rawOut.split('\n')) {
+    const trimmedLine = line.trim();
+    if (trimmedLine.startsWith('{') && trimmedLine.endsWith('}')) {
+      jsonOut = line;
+      break;
+    }
+  }
+
+  d('notarytool produced output:\n', output);
+
+  let parsed: any;
+  try {
+    parsed = JSON.parse(jsonOut);
+  } catch (err) {
+    throw new Error(`Could not parse notarytool output: \n\n${rawOut}`);
```

Comment:
Could we as a fallback try parse the entire `output` as JSON so that anything that worked before due to strange chunking or whatever keeps working.

Super appreciate the test coverage 🙇